.github/workflows/pr-gpu.yaml

```diff
@@ -0,0 +1,33 @@
+name: PR GPU tests
+on:
+  pull_request_target:
```

Comment:
same concern about secrets

Reply:
This unfortunately does need `pull_request_target` since the MCloud API secret needs to be passed.  In addition to 1) and 2) previously, the other safeguard in this case is that malicious code is actually sandboxed within the CI cluster so the damage is contained.  We do not pass any GH permissions to the code running on the MCloud CI cluster so this is probably fine.